validator/sawtooth_validator/journal/journal.py

```diff
@@ -38,15 +38,23 @@ def __init__(
         fork_cache_keep_time=300,  # seconds
         data_dir=None,
         observers=None,
+        genesis_observers=None,
```

Comment:
Typo in commit message: s/Cylinder/Cylinder signer/

Reply:
Fixed